components/support/base/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation Dependencies.kotlin_stdlib
     implementation Dependencies.kotlin_coroutines
 
+    api project(":concept-base")
```

Comment:
Should `concept-engine` (and other modules) depend on `concept-base` directly? With this they're depending on it transitively `concept-engine` -> `support-base` -> `concept-base`, which I guess is fine too. I think I would prefer the former, to make it explicit. Not sure, wdyt?

Reply:
Yes, they should. This line here is so that consumers of `support-base` do not explicitly import the interfaces too, that seemed unnecessary.

When I wanted to make `concept-engine` depend on `concept-base` I noticed that it already depends on `support-base` (which is exactly what I wanted to avoid) :(

At least `Observable` (that could be moved) and `ObserverRegistry` (which shouldn't be moved to the concept) are causing this.

I don't see a quick fix for this situation. In the long term we may be able to untangle this once the engine can depend on the store and dispatch directly (and therefore doesn't need to be observable independently).